delius-auto-test/sub-projects/spg.tfvars

```diff
@@ -12,8 +12,8 @@ route53_hosted_zone_id = "Z3VDCLGXC4HLOW"
 eng-remote_state_bucket_name = "tf-eu-west-2-hmpps-eng-dev-remote-state"
 
 # ENVIRONMENT ROLE ARNS
-eng_role_arn = "arn:aws:iam::895523100917:role/terraform"
-
+eng_role_arn = "${TERRAGRUNT_IAM_ROLE}"
```

Comment:
What are you expecting the value of variable "eng_role_arn" to be?
It will now literally be "${TERRAGRUNT_IAM_ROLE}" not the value of the export in the 
delius-auto-test.properties of 
export TERRAGRUNT_IAM_ROLE="arn:aws:iam::431912413968:role/terraform"